src/main/java/MessageManager.java

```diff
@@ -0,0 +1,49 @@
+public class MessageManager {
+
+    // Prints newest task
+    public static void printNewTask(Task task, int size) {
+        System.out.println("Got it. I've added this task:\n " +
+                taskToString(task) +
+                "\nNow you have " + size + " tasks in the list.");
+    }
+
+    // Converts task to string
+    public static String taskToString(Task task) {
+        return "[" + task.getType() + "][" + (task.isDone() ? "X" : " ") + "] "
+                + task.getTask() + (task.getType() == 'T' ? "" : " (" + (task.getType() == 'D' ? "by: " : "at: ")+ task.getTrail() + ")");
```

Comment:
As mentioned before, avoid ternary operators. Also, consider extracting out magic strings like 'T' and 'D' to constants.